source/Planet.cpp

```diff
@@ -231,6 +233,28 @@ void Planet::Load(const DataNode &node)
 
 
 
+void Planet::FinishLoading(Set<Wormhole> &wormholes)
+{
+	// If this planet is in multiple systems, then it is a wormhole.
+	if(!wormhole && systems.size() > 1)
+	{
+		wormhole = wormholes.Get(name);
+		wormhole->LoadFromPlanet(this);
+		Files::LogError("Warning: auto generation of wormhole \"" + name
+				+ (wormhole->IsValid() ? "\"." : "\" failed."));
+	}
+
+	// If the wormhole was autogenerated we need to update it to
+	// match the planet's state.
+	if(wormhole && wormhole->IsAutogenerated())
+	{
+		wormhole->SetLinked(!description.empty());
```

Comment:
SetLinked() and GenerateLinks() are actually already handled in LoadFromPlanet().

Reply:
Same comment as above. This is needed when a planet is updated.